server/http/wish/wish.go

```diff
@@ -0,0 +1,96 @@
+package wish
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/constants"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+)
+
+type wish struct {
+	apps *apps.Service
+}
+
+func Init(router *mux.Router, apps *apps.Service) {
+	w := wish{
+		apps: apps,
+	}
+
+	subrouter := router.PathPrefix(constants.WishPath).Subrouter()
+	subrouter.HandleFunc("", w.handleWish).Methods("POST")
+}
+
+func (wish *wish) verifyCallContext(actingUserID string, ctx apps.CallContext) error {
+	if ctx.ChannelID == "" {
```

Comment:
A channel ID is not always needed IMO. For example, a element on the main menu is not associated to any channel.

Reply:
@larkox It's likely not needed in the case of the main menu but we still have it available to provide in all cases, and it allows us to know what team they're currently viewing, may not be needed for a given request either of course. I think it's better to provide as much information as possible (like the channel id) for all calls, especially when it's not expensive.

Reply:
My worry is that since here we are requiring it, if we are in some context that does not make sense having one (the backstage for example, if we ever use it), it will create problems. I am all in for if we have a channelID, to check if we are part of that channel, but not requiring the channelID to be different to "".